tee-worker/cli/demo_direct_call_2_workers.sh

```diff
@@ -15,18 +15,12 @@ while getopts ":p:A:B:u:W:V:C:" opt; do
         A)
             WORKER1PORT=$OPTARG
             ;;
-        B)
-            WORKER2PORT=$OPTARG
-            ;;
         u)
             NODEURL=$OPTARG
             ;;
         V)
             WORKER1URL=$OPTARG
             ;;
-        W)
-            WORKER2URL=$OPTARG
-            ;;
```

Comment:
I think we can still keep these options - not each arg needs to be populated :)

Otherwise you'll have to modify it in every `cli/*.sh`

Reply:
Restored ✅